--- lzip-1.3/doc/lzip.texinfo~	2008-12-22 14:15:21.000000000 +0100
+++ lzip-1.3/doc/lzip.texinfo	2008-12-22 14:17:19.000000000 +0100
@@ -8,9 +8,9 @@
 @set UPDATED 21 December 2008
 @set VERSION 1.3
 
-@dircategory Data Compression
+@dircategory File utilities:
 @direntry
-* Lzip: (lzip).                 A data compressor based on the LZMA algorithm.
+* Lzip: (lzip).				A data compressor based on the LZMA algorithm
 @end direntry
 
 
