--- lzip-1.15/doc/lzip.texinfo.orig	2013-11-24 18:54:25.488920451 +0100
+++ lzip-1.15/doc/lzip.texinfo	2013-11-24 18:57:48.498911931 +0100
@@ -9,9 +9,9 @@
 @set UPDATED 20 September 2013
 @set VERSION 1.15
 
-@dircategory Data Compression
+@dircategory File utilities:
 @direntry
-* Lzip: (lzip).                 LZMA lossless data compressor
+* Lzip: (lzip).				LZMA lossless data compressor
 @end direntry
 
 
